src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.h

```diff
@@ -7,11 +7,21 @@ struct EnTakaraya;
 
 typedef void (*EnTakarayaActionFunc)(struct EnTakaraya*, PlayState*);
 
+#define EN_TAKARAYA_GET_SWITCH_FLAG(thisx) (u8)((thisx)->params >> 0x8)
+#define EN_TAKARAYA_GET_FF(thisx) ((thisx)->params & 0xFF)
+
 typedef struct EnTakaraya {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x164];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[24];
+    /* 0x218 */ Vec3s morphTable[24];
     /* 0x2A8 */ EnTakarayaActionFunc actionFunc;
-    /* 0x2AC */ char unk_2AC[0x10];
-} EnTakaraya; // size = 0x2BC
+    /* 0x2AC */ u8 eyeTexIndex;
+    /* 0x2AD */ u8 unk2AD;
+    /* 0x2AE */ s16 timer;
+    /* 0x2B0 */ s16 unk2B0;
+    /* 0x2B2 */ s16 switchFlag;
+    /* 0x2B4 */ Vec3s headRot;
+} EnTakaraya; /* size = 0x2BC */
```

Comment:
```suggestion
} EnTakaraya; // size = 0x2BC
```
Careful in changing the style of this comment